documentation/source/library-reference/system/file-system.rst

```diff
@@ -407,6 +408,49 @@ File-System module.
 
      - :func:`delete-directory`
 
+.. generic-function:: create-symbolic-link
+
+   Creates a symbolic link
+
+   :signature: create-symbolic-link *target* *link* => ()
+
+   :param target: An instance of `<pathname>`:class:.
+   :param link:   An instance of `<pathname>`:class:.
+
+   :description:
+
+     Creates a symbolic link to *target*.
+
+   :example:
+
+     Creates a symbolic link named *whattimeislove* to the *date* 
+     utility.
+
+     .. code-block:: dylan
+
+        create-symbolic-link("/usr/bin/date", "/tmp/whattimeislove"); 
+
+.. method:: create-symbolic-link
+   :specializer: <pathname>, <pathname>
+   :no-contents-entry:
+
+   :parameter target: An instance of :class:`<pathname>`.
+   :parameter link:   An instance of :class:`<pathname>`.
+
+.. method:: create-symbolic-link
+   :specializer: <string>, <string>
+   :no-contents-entry:
+
+   :parameter target: An instance of :drm:`<string>`.
+   :parameter link:   An instance of :drm:`<string>`.
+
+.. method:: create-symbolic-link
+   :specializer: <file-system-locator>, <file-system-locator>
+   :no-contents-entry:
+
+   :parameter target: An instance of :class:`<file-system-locator>`.
+   :parameter link:   An instance of :class:`<file-system-locator>`.
```

Comment:
Doc needs updating depending on how you resolve my comments below about these methods.